plugins/woocommerce/tests/php/src/Internal/DataStores/Orders/OrdersTableQueryTests.php

```diff
@@ -351,16 +351,22 @@ public function test_pre_query_escape_hook_pass_limit() {
 	 * @testdox A regular query will still work even if the pre-query escape hook returns null for the whole 3-tuple.
 	 */
 	public function test_pre_query_escape_hook_return_null() {
+		add_filter( 'woocommerce_hpos_pre_query', '__return_null', 10, 3 );
```

Comment:
I don't think it really matters, but we could simplify this to:

```suggestion
		add_filter( 'woocommerce_hpos_pre_query', '__return_null' );
```